### PR TITLE
Honor padded prop for InnerContent

### PR DESCRIPTION
**Pull request by kevinrademan**
IMPROVEMENT: Honor padded prop for InnerContent
IMPROVEMENT: Ensure we always publish to npm
FIX: Linting

### CODE REVIEW

**Comment by kevinrademan** [comment]
[Polish and tweaks](https://trello.com/c/UwkIhps2/28-polish-and-tweaks)

**Comment by jscheel** [comment]
@kevinrademan I'm concerned about the padding change. Have you check to see that it all still looks correct in the other iterations? I feel like there was a reason this changed..., maybe the branding stuff, or the median?

File: src/components/Container.js [hunk 1]
```diff
@@ -50,8 +50,10 @@ const StyledContent = styled(motion.div)`
 `
 
 const InnerContent = styled.div`
+  display: flex;
   flex: 1 1 auto;
-  padding-bottom: ${spacing.padding.small}px;
+  ${props =>
```
Comment (jscheel) [inline src/components/Container.js:55]: @kevinrademan I'm concerned about the display and padding change. Have you check to see that it all still looks correct in the other iterations? I feel like there was a reason this changed..., maybe the branding stuff, or the median?

Comment (kevinrademan) [reply src/components/Container.js:55]: I've gone through the kb, and contact and I couldnt find anything that looks out of place. To my knowledge the only place where this particular padded prop is getting used is actually in the chat system. All other parts of the app have it enabled

**Pull request merged**